sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/UnivocityGenerator.scala

```diff
@@ -60,6 +60,8 @@ class UnivocityGenerator(
     options.locale,
     legacyFormat = FAST_DATE_FORMAT,
     isParsing = false)
+  private lazy val nullAsQuotedEmptyString =
```

Comment:
nit: I think this can be a val as it's very likely to hit it. It's also relatively cheap so the lazy val protection may has bigger overhead.